src/components/common/ButtonBack/_ButtonBack.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../../styles/mixins';
+
+div.component-ButtonBack {
+  @include flex(row, start, space-arround);
+  margin: 10px 25px 0 10px;
+  .button-back-ico {
+    vertical-align: middle;
+    color: var(--input-raiting-text-color);
+    font-size: 1.4em;
```

Comment:
wielkość fontów masz podaną w variables ;)

Reply:
Poprawione. To był początek, i nie zauważyłem, że są zmienne, a później zapomniałem poprawić :)